Source/src/modules/ModuleSceneManager.cpp

```diff
@@ -166,16 +191,17 @@ bool Hachiko::ModuleSceneManager::CleanUp()
     EditorPreferences* editor_prefs = App->preferences->GetEditorPreference();
     editor_prefs->SetAutosave(scene_autosave);
 
-    #ifndef PLAY_BUILD
-    // If it was a temporary scene it will set id to 0 which will generate a new temporary scene on load
+#ifndef PLAY_BUILD
+    // If such scene was a temporary one it will set id to 0 which will
```

Comment:
I dont think we need this. We should be able to start with the real scene

Reply:
It's starting with the real scene actually.